### ISSUE TITLE
Total resistance not shown if two voltage sources exist in on sketch

### ISSUE THREAD

**Issue by K0rrid0r** [sharpie7/circuitjs1#594]
If two voltage sources exist in one sketch total resistance is not shown for any circuit.

![image](https://user-images.githubusercontent.com/56724286/116905737-ab318b00-ac3f-11eb-8a33-52f30014b96d.png)
everything like normal here.

![image](https://user-images.githubusercontent.com/56724286/116905799-c1d7e200-ac3f-11eb-8610-52adeaef0a26.png)
but what's up with this?

also why are these two circuits showing completely different values?
![image](https://user-images.githubusercontent.com/56724286/116905880-dcaa5680-ac3f-11eb-9de0-fa493a7ae0bb.png)

**Comment by pfalstad** [comment]
Yes, we only show resistance if there's only one voltage source in the circuit.  Because otherwise we'd have to analyze the circuit more carefully to determine whether the voltage sources are connected, and this feature didn't seem important enough to make that worth the effort.

These two circuits do not have explicit ground, so the simulator has to choose, and in this case it chose poorly.  Add a ground node to both, and then they will be the same: https://tinyurl.com/yfooogcp

**Comment by K0rrid0r** [comment]
has this been taken away in recent updates?

![image](https://user-images.githubusercontent.com/56724286/159140446-723a610f-1977-49f1-b335-3759369cdfbe.png)
